website/docs/docs/integrate/dbt-semantic-layer.md

```diff
@@ -0,0 +1,132 @@
+- Have both your production and development environments running dbt version 1.2 (latest) or higher
+- Use Snowflake data platfrom 
+- Install the metrics package version 0.3.2 or higher in your dbt project
+- Set up the [Metadata API](/docs/dbt-cloud/dbt-cloud-api/metadata/metadata-overview) in the integrated tool to import metric definitions
+- Recommended - Review the dbt metrics page and Getting started with the dbt Semantic Layer blog
+
+### Considerations
+
+Here are some important considerations to know about during public preview:
+
+- Support for Snowflake data platform only (_additional data platforms coming soon_)
+- Support for the deployment environment only (_development experience coming soon_)
+- Do not use environment variables for the job/environment (_coming soon_)
+
+## Public preview
+
+The dbt Semantic Layer is currently available for public preview, which means:
+
+&mdash; **Who?** The dbt Semantic Layer is open to all dbt Cloud tiers (Developer, Team, and Enterprise) during public preview. Developer accounts will be able to query the Proxy Server using SQL, but will not be able to browse dbt metrics in external tools, which requires access to the Metadata API. 
+
+&mdash; **What?** Public preview provides early access to new features, is supported and production ready, but not priced yet. Pricing for the dbt Semantic Layer will be introduced alongside the General Available (GA) release. There may also still be additions or modifications to product behavior. 
+
+&mdash; **When?** Public preview will end once the dbt Semantic Layer is available for GA. After GA, the dbt Semantic Layer will only be available to dbt Cloud **Team** and **Enterprise** plans.
+
+&mdash; **Where?** Public preview is enabled at the account level so you don’t need to worry about enabling it per user.
+
+&mdash; **Why?** Public preview is designed to test the functionality and collect feedback from the community on performance, usability, and documentation. 
+
+## Product architecture 
+
+The dbt Semantic Layer product architecture includes four primary components, and you can review that and its plan availability in more detail below:
+
+| Components | Information | Developer plans | Team plans | Enterprise plans | License |
+| --- | --- | :---: | :---: | :---: | --- |
+| **dbt metrics** | Allows you to define metrics in dbt Core. | ✅ | ✅ |  ✅  | Open source, Core |
+| **dbt Server**| HTTP server that is able to quickly compile metric queries per environment using dbt project code. | ✅ | ✅ | ✅ | BSL |
+| **SQL Proxy** | Reverse-proxy that accepts dbt-SQL (SQL + Jinja like query models and metrics, use macros), compiles the query into pure SQL, and executes the query against the data platform. | ✅ <br></br>_* Available during public preview only_ | ✅ | ✅ | Proprietary, Cloud (Team & Enterprise) |
```

Comment:
"Available during public preview only". What happens when the public preview is over?